test-resources/test-harness/infrastructure/template.yaml

```diff
@@ -20,6 +20,26 @@ Parameters:
     Description: "The stack containing the TXMA infrastructure"
     Type: String
     Default: txma-infrastructure
+  Environment:
+    Type: String
+    Default: dev
+    AllowedValues: [dev, localdev, build, staging]
```

Comment:
Shouldn't be allowed in staging as we're integrated with TxMA there

Reply:
Actually I guess if we start using it for other things (eg DDB) maybe we would want in staging will resolve this comment